components/basic_widgets/InputWidget.vue

```diff
@@ -101,6 +101,21 @@ export default {
       return this.disable_paste == true ? "return false;" : "";
     },
   },
+  created() {
+    this.unsubscribe = this.$store.subscribe(async (mutation) => {
+      if (
+        (this.dom_id_suffix === "heatmap-max" || this.dom_id_suffix === "heatmap-min") &&
```

Comment:
Not sure if this will matter, but I was thinking this sort of check would be helpful for retaining text entries in other input widgets. Not sure if these values are retained when switching between pages though, in that case this check isn't very important